src/main/kotlin/org/malachite/estella/people/api/HrPartnerController.kt

```diff
@@ -36,6 +39,15 @@ class HrPartnerController(@Autowired private val hrPartnerService: HrPartnerServ
         return ResponseEntity(partner, HttpStatus.OK)
     }
 
+    @CrossOrigin
+    @GetMapping("/{hrPartnerId}/offers")
+    fun getHrPartnerOffers(@PathVariable("hrPartnerId") hrPartnerId: Int): MutableList<OfferResponse> {
```

Comment:
To rozumiem takie tymczasowe bo tak w sumie id powinniśmy wziąć jak na moje z tokenu plus i wtedy możemy albo zweryfikować id ze ścieżką albo stworzyć funkcję w offerService getOffersByCreator(hrPartnerId:Int)

Reply:
Zadanie jest na tyle małe, że w sumie można poczekać na tokeny i zrobić od razu do porządku

Reply:
Nie przeszkadza, żeby były 2 takie endpointy